selfie-camera/src/main/res/layout/activity_capture_selfie.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2017 Nafundi
+<?xml version="1.0" encoding="utf-8"?><!-- Copyright 2017 Nafundi
```

Comment:
I don't know if it make sense to keep such license notes anymore. @lognaturel what do you think?

Reply:
Yeah good question. I'm never confident that I can remove these. We should probably make a decision on that and then add it to `CONTRIBUTING.md`.

Reply:
Discussed with @lognaturel, and we don't have a good answer about removing them or not so will leave this in for now.